manifests/profile/hathitrust/solr6/classic.pp

```diff
@@ -0,0 +1,27 @@
+# nebula::profile::hathitrust::solr6::classic
+#
+# Profile for the classic HathiTrust solr lss/catalog servers
+#
+# @example
+#   include nebula::profile::hathitrust::solr6::classic
+class nebula::profile::hathitrust::solr6::classic (
+  String $loki_endpoint_url = lookup('nebula::profile::hathitrust::loki_endpoint_url'),
+  Boolean $catalog_solr = false,
+){
+  if($catalog_solr) {
```

Comment:
I'm wondering if rather than having the conditionality here, it might make sense to provide a way to provide additional log files to ship via hiera? That would also make it easier to experiment with shipping additional log files on an individual host in the future without needing to add additional complexity/conditionality/more environments